tests/test_filepath.py

```diff
@@ -75,16 +75,15 @@ def test_duplicate_error(store="repo"):
     """ syncing duplicate non-matching file should fail """
     ext = schema.external[store]
     stage_path = dj.config['stores'][store]['stage']
-    filename = 'thesis.dat'
     relpath = 'one/two/three'
-    os.makedirs(os.path.join(stage_path, relpath), exist_ok=True)
-    managed_file = os.path.join(stage_path, relpath, filename)
-    with open(managed_file, 'wb') as f:
+    managed_file = Path(stage_path, relpath, 'thesis.dat')
+    managed_file.parent.mkdir(parents=True, exist_ok=True)
+    with managed_file.open('wb') as f:
         f.write(os.urandom(300))
-    ext.fput(managed_file)
-    with open(managed_file, 'wb') as f:
+    ext.upload_filepath(managed_file)
+    with managed_file.open('wb') as f:
         f.write(os.urandom(300))
-    ext.fput(managed_file)  # this should raise exception because the file has changed
+    ext.upload_filepath(managed_file)  # this should raise exception because the file has changed
 
 
 def test_duplicate_error_s3():
```

Comment:
This test failed on WIN10. For example, the result of `managed_file` was `dj/repo/one\two\three\thesis.dat`.